tests/CronJobTest.php

```diff
@@ -10,14 +10,14 @@
 use RenokiCo\PhpK8s\Kinds\K8sJob;
 use RenokiCo\PhpK8s\ResourcesList;
 
-class CronCronJobTest extends TestCase
+class CronJobTest extends TestCase
 {
     public function test_cronjob_build()
     {
         $pi = K8s::container()
             ->setName('pi')
-            ->setImage('perl')
-            ->setCommand(['perl',  '-Mbignum=bpi', '-wle', 'print bpi(2000)']);
+            ->setImage('public.ecr.aws/docker/library/perl')
+            ->setCommand(['perl',  '-Mbignum=bpi', '-wle', 'print bpi(200)']);
```

Comment:
The newer Perls were throwing errors on 2000 overflowing BigNumber or something.